### PR TITLE
[TRUS-2673]  - Successful claim an estate and successful verify identity

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: app/views/IvSuccessWithoutPlaybackView.scala.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ *@
+
+@import config.FrontendAppConfig
+
+@this(
+ main_template: MainTemplate,
+ config: FrontendAppConfig
+)
+
+@(isAgent: Boolean, utr: String)(implicit request: Request[_], messages: Messages)
```
Comment (jaynabonne) [inline app/views/IvSuccessWithoutPlaybackView.scala.html:24]: Is isAgent needed?

File: conf/application.conf [hunk 2]
```diff
@@ -73,22 +75,60 @@ microservice {
...
       }
 
+          tax-enrolments {
+            protocol = http
+            host = localhost
+            port = 9995
+          }
+
       features {
         welsh-translation: true
```
Comment (jaynabonne) [inline conf/application.conf:115]: welsh false? :)

Comment (jassalrichy) [reply conf/application.conf:115]: Set to false.

File: app/views/IvSuccessWithoutPlaybackView.scala.html (hunk 1 above)
Comment (jassalrichy) [reply app/views/IvSuccessWithoutPlaybackView.scala.html:24]: isAgent removed from this view.

**Pull request merged**